django/db/backends/base/base.py

```diff
@@ -36,6 +37,8 @@ class BaseDatabaseWrapper(object):
     ops = None
     vendor = 'unknown'
     SchemaEditorClass = None
+    # Classes instantiated inside __init__().
+    validation_class = BaseDatabaseValidation
```

Comment:
@timgraham See here for where the dummy backend customizes the other helper classes: https://github.com/django/django/blob/0bbab97c289ebcfd5676d4c84686c5549e650f8d/django/db/backends/dummy/base.py#L38